build/cloudbuild-automation/README.md

```diff
@@ -0,0 +1,67 @@
+### Building with CloudBuild - Automated Builds
+
+This repo shows how you can leverage Google Cloud Build triggers to automate your custom collector builds based on certain events on GitHub (or other similar version control repository hosting service).
+
+After you have followed instructions in this README, you should have the following automation setup - 
+ - You will have an [Artifact Registry](https://cloud.google.com/artifact-registry) setup in your GCP project where all your custom opentelemetry collector docker images will be published. 
+ - You will have the GitHub repository (containing the code to build your custom collector) linked with a [Cloud Build Trigger](https://cloud.google.com/build/docs/triggers) which will be configured to trigger an automated build for a collector image whenever there is a push to the `main` branch of the repository (*this can be updated to track any branch*).
```

Comment:
`You will have the GitHub repository (containing the code to build your custom collector) linked with...` is this a new Github repository? Maybe say explicitly `You will have a new GitHub repository...`

Reply:
"New" repository might mean a brand new repository since the instructions have not explicitly mentioned forking this repo, should I instead write - 
`You will have your Github repository (*Containing the code to build your custom collector*) linked with a Cloud Build trigger` ?